### ISSUE TITLE
enhance documentation for custom payloads

### ISSUE THREAD

**Issue by marina-p** [microsoft/restler-fuzzer#135]
As discussed in related issue #132, it would be useful to have more detailed examples of the different types of supported custom payloads.  Currently, only the basic syntax is described in FuzzingDictionary.md

**Comment by efebarlas** [comment]
Hi, I am working on a project that would benefit a lot from custom payloads. Unfortunately, I was not able to get it to work from the existing documentation. Can you add more documentation for this issue when you have the time?

**Comment by marina-p** [comment]
Hello @efebarlas,

Yes, we will add the documentation.  In the meantime, do the following examples unblock you?

The json pointer refers to the json body type of the response only.  If you have two different endpoints with the same body json type, and need to specify different custom payloads for them, you will have to add them both to the same list in the dictionary, and both endpoints will test them.  You cannot refer to a nested property by pointer, it has to start from the root of the actual request schema of one of the endpoints.  

Example (from one of the unit tests checked into the repo): one of the body's properties is an array "items", where each entry is of type "GroceryListItem", and you want to specify "deliveryTags" in the custom payload.

```
        "items": {
          "description": "The type of bags to use",
          "type": "array",
          "items": {
            "$ref": "#/definitions/GroceryListItem"
          }
        }
```

```
    "GroceryListItem": {
      "properties": {
        "name": {
          "description": "The name of the item",
          "type": "string"
        },
        "deliveryTags": {
          "description": "The delivery tags",
          "type": "object"
        }

```

```
"restler_custom_payload": {
     "/items/[0]/deliveryTags": [ "custom1" ]
}
```

**Comment by efebarlas** [comment]
Yes, these examples were very helpful. I will look at other custom payload unit tests as well. Thank you!
